pkgs/applications/display-managers/sddm/default.nix

```diff
@@ -41,11 +41,11 @@ in mkDerivation rec {
     # not supported anyway.
     "-DUID_MIN=1000"
     "-DUID_MAX=29999"
-  ];
 
-  preConfigure = ''
-    export cmakeFlags="$cmakeFlags -DQT_IMPORTS_DIR=$out/$qtQmlPrefix -DCMAKE_INSTALL_SYSCONFDIR=$out/etc -DSYSTEMD_SYSTEM_UNIT_DIR=$out/lib/systemd/system"
-  '';
+    "-DQT_IMPORTS_DIR=${placeholder "out"}/${qtbase.qtQmlPrefix}"
+    "-DCMAKE_INSTALL_SYSCONFDIR=${placeholder "out"}/etc"
+    "-DSYSTEMD_SYSTEM_UNIT_DIR=${placeholder "out"}/lib/systemd/system"
+  ];
```

Comment:
Isn't there a `cmakeFlags` attribute available? Not sure though!

Reply:
GitHub seemed to collapse the diff in a weird way. Above this is in `cmakeFlags`.